Deliverables/Submitted/D2 - Requirements.md

```diff
@@ -0,0 +1,341 @@
+
+### 1.3 Value proposition and customer segment
+
+Value prop: Newstr is a free and easy-to-use application that allows people who struggle to find pertinent topics to receive news that suits their interests easily and daily through email.
+
+Consumer segment: Everyday people who struggle to find interesting and relevant news stories.
+
+## 2. Stakeholders
+
+**Project Manager** - Zachary Parham
+
+Responsible for assigning responsibilities, and managing deadlines and resources to ensure timely completion of assigned tasks. 
+
+**Application Developers** - “The Fellaz” ( Jackson Brittain, Zachary Parham, Hyunsoo Kim, William Rogers, Jacob Olson)
+
+Develop the required tasks of the software. Developers should also test, debug and improve useability and reliability of software (web applications and mobile applications).
+
+**Testers/ Users** - NAU classmates and peers, The Interviewees, “The Fellaz”
+
+Testers/Users will try our software and give us helpful feedback
+ to help developers improve usability, reliability, and functionality.
+
+**Competitors** - NY Times, CNN, The Heralds, FOX, NBC, MSN, Apple News
+
+Other news applications which provide similar functionality to the users. We will have similar functionality, but we will have unique features to set us apart.
+
+## 3. Functional requirements (features)
+1. Send news to users
+    - users will receive news from their inbox
+2. Customizable topics
+    - users can pick news topics they are interested in
+3. Variable email frequency
+    - users can pick a time and a frequency to receive the news
+4. Sort news by bias
+    - users can choose the bias of political news 
+5. Customizable news sources
+    - users can choose what sources they prefer
+6. Stores user emails
+    - backend will store emails and send news automatically
+
+
+## 4. Non-functional requirements
+Reliability - 95% of users receive their news at the sending time
+Compatibility - Users accessing both the  mobile app and web app receive the same information
+Usability - Users won't have any trouble setting up accounts
+Portability - Users can view news from their phone
+Security - User information will be stored securely and will use security protocols during registration
+
+
+## 5. MVP
+**Features:**
+- Send news to users
+- Customizable news topics
+- Variable email frequency
+
+**Description:**
+	The MVP will be a smaller app that allows users to select news topics that interest them
+
+  
+## 6. Use cases
+### 6.1 Use case diagram
+![Use Case Diagram](https://user-images.githubusercontent.com/71902810/193481228-f57c2553-d022-409c-8f0b-72eda95a5067.png)
+
+### 6.2 Use Case Descriptions and interface Sketch
```

Comment:
review the use case descriptions to use the format of dialog between users and system.